lecture/src/main/docker/jib/entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+echo "The application will start in ${JHIPSTER_SLEEP}s..." && sleep ${JHIPSTER_SLEEP}
+exec java ${JAVA_OPTS} -noverify -XX:+AlwaysPreTouch -Djava.security.egd=file:/dev/./urandom -cp /app/resources/:/app/classes/:/app/libs/* "de.tum.in.www1.artemis.lecture.LectureApp"  "$@"
```

Comment:
Codacy

Reply:
This is okay, since otherwise running on docker doesn't work